style.css

```diff
@@ -0,0 +1,64 @@
+
+.textcontainer {
+  display: flex;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+}
+
+.secondimagecontain {
+  background-image: url("image_geometry_2-1.svg");
+  display: flex;
+  width: 100%;
+  height: 160px;
+  top: 330px;
+  position: relative;
+}
+
+.text1 {
+  position: absolute;
+  width: 220px;
+  height: 100px;
+  left: 16px;
+  top: 70px;
+  font-family: 'Playfair Display', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 50px;
+  line-height: 100%;
+  color: #1a2236;
+}
+
+.text2 {
+  position: absolute;
+  width: 320px;
+  height: 126px;
+  left: 16px;
+  top: 224px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 21px;
+  color: #1a2236;
+}
```

Comment:
- [ ] The requirement is that we use flexbox to align children of parent elements. Let's not use absolute positioning. Here you can easily give the parent container which is `.textcontainer` a flex-direction and other possible flex properties to align these children. See a screenshot of requirement below 👇🏼

    ![flex-box](https://user-images.githubusercontent.com/45002654/125559584-e932f8c0-4b45-40c6-8d3b-9b77e27b525c.PNG)

Reply:
I have placed the texts with flex property successfully. The changes are made as per your advice.